packages/sourcecred/src/plugins/discord/update_discord_api_snapshots.sh

```diff
@@ -15,7 +15,7 @@ if [ -z "${SOURCECRED_TEST_SERVER_TOKEN:-}" ]; then
   exit 1
 fi
 
-toplevel="$(git -C "$(dirname "$0")" rev-parse --show-toplevel)"
+toplevel="$(git -C "$(dirname "$0")" rev-parse --show-toplevel)/packages/sourcecred"
```

Comment:
Can you capture this as an environment variable at the top of the script? It's much more readable to hoist this sort of detail to the top of the file, and then invoke it where needed, even if it;s just once. 

We might have to reconsider how we organize our bash scripts so we can centralize details like this and make debugging, updates and refactors easier. 

How do you feel about storing this value at repo root and then appending it everywhere it's needed, rather than appending this raw string everywhere? Basically just create a `CORE_SUBPATH` variable with this string in a monorepo_vars.sh file that gets invoked everywhere.

Reply:
That sounds like a great idea I'll look into how to define reusable variables in bash and do that